### ISSUE TITLE
Panic connecting v2.1.1 VC to v2.0.1 or earlier BN

### ISSUE THREAD

**Issue by michaelsproul** [sigp/lighthouse#2958]
```
thread 'tokio-runtime-worker' panicked at 'called `Result::unwrap()` on an `Err` value: FromHexError { inner: InvalidStringLength }', consensus/types/src/chain_spec.rs:694:10
```

See https://github.com/sigp/lighthouse/pull/2957 for more information

**Comment by mnguyen-io** [comment]
Hi, please see:

https://github.com/sigp/lighthouse/releases/tag/v2.1.0

**Breaking Changes
The VC from this release (v2.1.0) is not compatible with Lighthouse BNs from prior versions.
Users MUST update both Lighthouse BNs and VCs to v2.1.0.**

**Comment by joe-bowman** [comment]
I was able to replicate this earlier too; what's the plan here, upgrade beacon nodes first?

**Comment by michaelsproul** [comment]
> I was able to replicate this earlier too; what's the plan here, upgrade beacon nodes first?

Yeah. In the release notes for v2.1.0 and v2.1.1 we signalled an intention for the v2.1.x validator _not_ to be compatible with the v2.0.1 BN, so the recommended upgrade path remains the same: upgrade the BNs, and then the VCs shortly after (or in the same update). This panic is just an unsightly error rather than a clean failure.

**Comment by joe-bowman** [comment]
Excellent, thank you for your swift response!

**Comment by michaelsproul** [comment]
Fixed in v2.1.2
